Modules/Loadable/Volumes/SubjectHierarchyPlugins/qSlicerSubjectHierarchyVolumesPlugin.cxx

```diff
@@ -145,6 +149,15 @@ void qSlicerSubjectHierarchyVolumesPluginPrivate::init()
   this->ResetViewOrientationOnShowAction->setCheckable(true);
   this->ResetViewOrientationOnShowAction->setChecked(false);
 
+  // Add color legend action
+
+  this->ShowColorLegendAction = new QAction("Show color legend", q);
```

Comment:
Mark the string translatable by using `tr`.

```suggestion
  this->ShowColorLegendAction = new QAction(tr("Show color legend"), q);
```

Reply:
> It looks good, I've just added a few nitpicks. Please also add a line to the documentation where the options for show/hide color legend is listed (https://slicer.readthedocs.io/en/latest/user_guide/modules/colors.html#show-color-legend).

Description added.